.vscode/settings.json

```
@@ -0,0 +1,12 @@
+// Document用設定ファイル
+{
+    // d2 のレイアウトエンジンの指定
+    "D2.previewLayout": "elk",
+    "[d2]": {
+        // d2ファイルのタブスペース数の指定
+        "editor.tabSize": 4
+    },
+    "[markdown]": {
+        "editor.tabSize": 2,
```

こちらも

まあタブ文字がいいかスペースがいいかは言語側で指定がない限り好みな気がするのでよしなにしちゃってください :pray:

```suggestion
        "editor.tabSize": 2,
        "editor.insertSpaces": true
```

---

タブ文字混入は僕も好かないので、取り込みました